util/vendor.py

```diff
@@ -415,6 +428,23 @@ def take_path(p):
                                 'exclude_from_upstream has entry {}, which is not a string.'
                                 .format(efu))
 
+    def apply_overrides(self, desc_data, desc_overrides):
+        """ Apply overrides from command line to configuration file data
+
+        Updates are applied to the desc_data reference."""
+
+        # Create a full desc-like structure of overrides.
+        for key, value in desc_overrides:
+            log.info("Overriding description key '{}' with value '{}'".format(
```

Comment:
Really minor nit, but I'd consider using `{!r}` instead of putting in the quotes manually. It means that you get a more helpful message if value is something like `foo'bar`.